other-docs/guides/launching-a-site-on-altis.md

```diff
@@ -0,0 +1,104 @@
+
+Once you've collected all the information into your GitHub issue, this information can be copied into [a support ticket in the Altis Dashboard](https://docs.altis-dxp.com/guides/getting-help-with-altis/) to alert the Altis Support Team. Set the ticket Type to a "Task", the Priority to "Normal" and the Subject can be "Site Launch Preparation".
+
+### Example launch checklist
+
+```markdown
+# Goal
+
+<!-- A brief overview of the launch objective. Is it a migration? Is it a new site? -->
+
+## Pre-launch checklist
+<!-- To be filled out by the project team/lead. Not needed for Altis Support ticket. -->
+- [ ] Are we deploying the correct branch?
+- [ ] Has AWS SES been setup?
+- [ ] Is a content freeze active?
+- [ ] Is the site ready for the DNS switch?
+- [ ] Are any redirections needed?
+
+## Domains hosted on Altis
+<!-- List of domains that will be on the stack. Dev and staging environments do not need to be included in the list. -->
+
+- domain.com
+- domain2.com
+- etc.
+
+**What domain(s) will email need to be sent from?**
+<!-- Enter as a list -->
+- noreply@domain.com
+- etc.
+
+**For each site in the network, what will be the FROM email?
+<!-- Enter as a list -->
+- **Network Admin:** noreply@domain.com
+- **domain.com:** some-email@domain.com
+- etc.
+
+## Environment
+* **Database import link:**
+* **Uploads import link:**
+* **Who is providing the SSL certificate(s) (Altis or client)?:** 
+* **Who controls the DNS for the client (Altis or client)?:**
+* **Projected traffic:** 
+* **Any other specifications or challenges?**
+_Refer to the Altis [Limitations](https://docs.altis-dxp.com/cloud/limitations/) and [Page Caching](https://docs.altis-dxp.com/cloud/page-caching/) guides for specifics around what requests can be supported._
+
+
+```
+
+## Preparing Altis Cloud Environments
+After you have submitted your support request in the Altis Dashboard, the Altis Support team will work on setting up your environments and be in touch with your team. When your new cloud environments are set up, you will be contacted and can begin deploying your code to those new environments.
+
+Once you have environments set up, you should restrict access to them by [requiring login](https://docs.altis-dxp.com/security/require-login/), forcing [PHP basic authentication](https://docs.altis-dxp.com/security/php-basic-auth/), or both. In your Altis configuration file (`composer.json`), add the following to require login both and PHP authentication. You may omit PHP authentication, but we recommend having at least one of these enabled for all non-production and pre-launch sites. You can override the require login setting to not require login on local environments. PHP authentication is disabled by default on local environments. Refer to the documentation pages for more information about Require Login and PHP Basic Authentication.
```

Comment:
I would rephrase this to break it up a bit. Also, it's not "PHP authentication", it's "HTTP Basic authentication" (or just "Basic auth"); that might be something we need to correct in the other docs too?

```suggestion
Once you have environments set up, we recommend limiting access to your sites. By default, login will be required on your development and staging environments. Before your site launches, we also recommend requiring login for your production environment as well.

Logins can be enforced by [requiring user logins](https://docs.altis-dxp.com/security/require-login/), forcing [HTTP basic authentication](https://docs.altis-dxp.com/security/php-basic-auth/), or both.

In your Altis configuration file (`composer.json`), add the following to both require login and HTTP Basic authentication. You may omit Basic authentication, but we recommend having at least one of these enabled for all non-production and pre-launch sites. You can override the require login setting to not require login on local environments. Basic authentication is disabled by default on local environments. Refer to the documentation pages for more information about Require Login and Basic Authentication.
```

It seems to me like we should also just enable require-login on prod, so that when the site launches, it's easy to just remove those lines?

Reply:
> It seems to me like we should also just enable require-login on prod, so that when the site launches, it's easy to just remove those lines?

Could do. I can spin up an issue for it.